geemusic/__init__.py

```diff
@@ -10,10 +10,12 @@
 app = Flask(__name__)
 ask = Ask(app, '/alexa')
 
-if str(environ['DEBUG_MODE']) is True:
+if str(environ['DEBUG_MODE']) == 'True':
```

Comment:
Does `environ['DEBUG_MODE']` evaluate to `True`? If so, it's likely better to remove the `str` conversion rather than test on string equality.

Reply:
os.environ should be all strs.